tests/test_sqlite.py

```diff
@@ -15,9 +15,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'file:memorydb_default?mode=memory&cache=shared',
     },
     'other': {
         'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'file:memorydb_other?mode=memory&cache=shared',
```

Comment:
Could you elaborate on why this is needed?

Reply:
I did so in [this comment](https://github.com/django/django/pull/11452#discussion_r298794858).

Reply:
It should not be necessary anymore.